feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/syncinfo/usecase/ObserveSyncInfoUseCase.kt

```diff
@@ -169,6 +170,12 @@ internal class ObserveSyncInfoUseCase @Inject constructor(
             else -> OnStandby
         }
 
+        val isUpSyncStillAllowedWhenCommCareSyncBlockedByDeniedPermission = // quite a specific case hence the name
+            isCommCareSyncBlockedByDeniedPermission && projectConfig.canCoSyncData()
```

Comment:
Actually you can have CommCare "down" sync and Simprints up sync. Down and upsync configurations are independent of each other. Down can be **either** Simprints or CommCare while up can be any (or all) of Simprints and CoSync. However, CoSync upsync is a bit of a misnomer since data is not really upsynced to an endpoint but rather returned in Intent result.